infra/terraform/external_function/snowalert/lambda.tf

```diff
@@ -10,7 +10,7 @@ data "archive_file" "lambda_code" {
 }
 
 resource "aws_lambda_function" "stdefn" {
-  function_name    = "${var.prefix}_external_function"
+  function_name    = var.aws_lambda_function_name
```

Comment:
this one seems all right as standard -- let's not make it a variable, unless there is a reason to?